In [1]:
import numpy as np
import pandas as pd
import scipy.stats as stats
import matplotlib.pyplot as plt
import seaborn as sns



1. It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

    + Set up the hypothesis test.
    + Write down all the steps followed for setting up the test.
    + Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can       you make based on this calculated value?



In [8]:
mean=120
n=100 #
X=130.1
standard_dev=21.21
mean=120

In [9]:
#test statistic for a Z-test is the Z-statistic, which has the standard normal distribution under the null hypothesis
#z = ( sample_mean - pop_mean ) / ( pop_std_dev / np.sqrt(45) )
#z=X-mittelwert/standardabweichung

z = ( X - mean )/( standard_dev / np.sqrt(n))

In [13]:
print("test statistic:", z)

test statistic: 4.761904761904759


In [22]:
zc = stats.norm.ppf(0.95)
zc

#The critical value corresponding to a 0.95 area of a normal distribution is: 1.64

1.6448536269514722

In [23]:
zc = stats.norm.ppf(1-(0.05/2))
zc

1.959963984540054

What decision can you make based on this calculated value?

there must be a significant difference from the regular population because HO is rejected 

#z= 4.761904761904759 > 1.6444 -> reject H0 (if z<1.96 -> H0 not rejected)

In [ ]:
zc = stats.norm.ppf(0.98)